packages/nitro-protocol/gas-benchmarks/gas.ts

```diff
@@ -17,7 +17,12 @@ type Path =
   | 'ETHexitSadLedgerFunded';
 
 // The channel being benchmarked is a 2 party null app funded with 5 wei / tokens each.
-
+// KEY
```

Comment:
These diagrams are straightforward but quite useful.

Should each step be included inside the relevant implementing test at the appropriate spots?

Reply:
I've got some improvements coming to these diagrams in a stacked PR. I think putting them in the implementation is also a good idea -- I'll take a look at that in a follow-up PR.